### PR TITLE
#139284023 CRUD operations SPM

### PR DESCRIPTION
**Pull request by jkanyua**
#### What does this PR do?
 - Fixes getting started video bug
 - Adds implementation of (not-so) static pages. Create pages in backend, have them appear in front

 #### Description of Task to be completed?
Create and edit static pages, such as Contact, About, or Support.
Implement static pages for a given shop.
Enable admins to customize the look on the static page content

 #### How should this be manually tested?
- Log in as admin
- Add page
- Check navigation bar for the route to the page

 #### Any background context you want to provide?

 #### What are the relevant pivotal tracker stories?
#139284023 

 #### Screenshots (if appropriate)
image
image
image

 #### Questions:

### CODE REVIEW

File: client/templates/index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <head>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <script src="//cdn.ckeditor.com/4.6.2/standard/ckeditor.js"></script>
```
Comment (collins-b) [inline client/templates/index.html]: Are you sure this link is working?It should be https://cdn.ckeditor.com/4.6.2/standard/ckeditor.js

File: imports/plugins/included/default-theme/client/styles/static-pages/static-pages.less [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.new_page {
+  margin-bottom: 60px;
+}
+#page-content {
+  padding-top: 20px;
+}
+.static-pages-dropdown {
+  padding: 0 15px;
+  // z-index: 2000;
```
Comment (collins-b) [inline imports/plugins/included/default-theme/client/styles/static-pages/static-pages.less]: Is there a reason why you have this commented code here?

File: imports/plugins/included/static-pages/client/templates/static-pages.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+           <input type="text" class="form-control" name="title" id="static-page-title" placeholder="Please enter a page title">
+         </div>
+         <div class="input-group">
+           <span class="input-group-addon" id="basic-addon3">{{baseUrl}}/pages/</span>
+           <input type="text" class="form-control" id="static-page-slug" aria-describedby="basic-addon3" placeholder="URL/SLUG">
+         </div>
+         <h4>Page Content:</h4>
+         <textarea cols="20" rows="10" name="body" id="static-page-content"></textarea>
+         <script>
+
```
Comment (collins-b) [inline imports/plugins/included/static-pages/client/templates/static-pages.html]: Do you have a reason for this here?

File: imports/plugins/included/static-pages/client/templates/static-pages.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
+import { Template } from "meteor/templating";
+ import "./static-pages.html";
+ import { StaticPages } from "/lib/collections";
+ import { Reaction } from "/client/api";
+ 
+ Template.staticPages.onCreated(() => {
+    // Subscribe to Pages publication
```
Comment (collins-b) [inline imports/plugins/included/static-pages/client/templates/static-pages.js]: Single-line comments not preferred.Try to us JSDoc

File: imports/plugins/included/static-pages/register.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import { Reaction } from "/server/api";
+
```
Comment (collins-b) [inline imports/plugins/included/static-pages/register.js]: Why two empty lines instead of one?

File: server/security/policy.js [hunk 6]
```diff
@@ -51,3 +51,7 @@ BrowserPolicy.content.allowOriginForAll("enginex.kadira.io");
 BrowserPolicy.content.allowOriginForAll("*.stripe.com");
 BrowserPolicy.content.allowOriginForAll("http://*.paystack.co");
 BrowserPolicy.content.allowOriginForAll("https://paystack.com");
+
```
Comment (collins-b) [inline server/security/policy.js]: Why have you introduced this empty line here?

**Review by collins-b — CHANGES_REQUESTED** [review]

File: client/templates/index.html (hunk 1 above)
Comment (jkanyua) [reply client/templates/index.html]: As explained on slack:
A relative URL without a scheme (http: or https:) is valid, per RFC 3986: "Uniform Resource Identifier (URI): Generic Syntax", Section 4.2.

**Review by jkanyua — COMMENTED** [review]
@andela-cwekesa Please review changes

**Review by collins-b — APPROVED** [review]

**Pull request merged**